components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -117,6 +117,11 @@ enum RuntimeAgentConnectionStatus {
     Disconnected
 }
 
+type CleanUpRuntimeStatus {
+    id: String!
+    message: String
```

Comment:
I think naming could be a little more accurate. I am not sure if `Status` suffix is appropriate here.